src/helpers/files.js

```diff
@@ -1,4 +1,4 @@
-//  @ts-check
+/* eslint-disable max-lines */
```

Comment:
[sand] We should probably avoid disabling eslint and separate this file into smaller ones moving forward